releasenotes/notes/bump-go-to-1.13.11-de62f7d057ea740b.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    The Agent, Logs Agent and the system-probe are now compiled with Go ``1.13.11``.
```

Comment:
I used the same message as https://github.com/DataDog/datadog-agent/pull/5167.

Reply:
@ogaca-dd #5167 actually changed the release note to mention `all agent binaries` instead of just `agent, logs and system-probe`, so unless this has changed since then, let's use the broader meaning for accuracy